recipes/xo/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.1.11" %}
+
+package:
+  name: xo
+  version: {{ version }}
+
+source:
+  fn: xo.tar.gz
+  url: https://github.com/scopatz/xo/archive/8f486a2709d376d00f98c09a86179bce2a99099a.tar.gz
```

Comment:
I guess we would typically also add a sha256 to assert that somebody malicious hasn't man-in-the-middled you, but it isn't a deal breaker. If you also put the version in the filename, that would stop conda build from thinking it already has the file in its cache. i.e. this section becomes:

```
source:
  fn: xo-{{ version }}.tar.gz

  url: https://github.com/scopatz/xo/archive/8f486a2709d376d00f98c09a86179bce2a99099a.tar.gz
```